README.md

```diff
@@ -57,6 +57,12 @@ method: post
 ---
 {% liquid
   assign total = 0
+  
+  comment
+    DO NOT USE THIS CODE IN PRODUCTION - Never trust user input, the selected line items and their details, like price
+    should be fetched server side from the database. This code is for demonstration purposes only.
+  endcomment
+
```

Comment:
@Slashek Instead of adding scary comment I would change the example so it gets the price from some array defined in the liquid.